spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -110,4 +110,48 @@ module MiqAeServiceSpec
       end
     end
   end
+
+  describe MiqAeService do
+    context "service models" do
+      it "expose all expected active_record models as service_models" do
+        excluded_model_names = %w(
+          AuthToken
+          AuthUseridPassword
+          Category
+          Datacenter
+          ManageIQ::Providers::BaseManager
+          ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job
+          VmServer
+          VmSynchronize
+        )
+
+        # Determine base_model for all service model files
+        base_models = Dir.glob(MiqAeMethodService::MiqAeServiceModelBase::SERVICE_MODEL_GLOB).collect do |f|
+          model_name = MiqAeMethodService::MiqAeServiceModelBase.model_name_from_file(f)
+          next if model_name == "MiqAeServiceMethods"
+
+          "MiqAeMethodService::#{model_name}".constantize.send(:model).base_model
+        end.compact!.uniq!
```

Comment:
Should this also be a method in the code somewhere?  It doesn't feel like test code.